### PR TITLE
Different sugar

### PR DESCRIPTION
**Pull request by safarinarges**

### CODE REVIEW

File: cake-recipe.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canols oil
```
Comment (ahaiderh) [inline cake-recipe.txt]: maybe check the oil name :)

**Review by ahaiderh — CHANGES_REQUESTED** [review]
just a small change for oil name

**Comment by ahaiderh** [comment]
final comment

File: cake-recipe.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+ - 2 cups brown sugar
  - 1 3/4 cups all-purpose flour
  - 3/4 cup cocoa
  - 1 1/2 teaspoons baking powder
  - 1 1/2 teaspoons baking soda
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canola oil
```
Comment (safarinarges) [inline cake-recipe.txt]: Fixed the typo.

**Pull request merged**